open-machine-learning-jupyter-book/data-science/working-with-data/pandas.md

```diff
@@ -3525,4 +3518,6 @@ Here are some examples of exploring data from Image data sources:
 
 ## Acknowledgments
 
+Thanks for the [introduction of Pandas by Wikipedia](https://en.wikipedia.org/wiki/Pandas_(software)).
```

Comment:
You should add a {cite} inline for this. You can search {cite} in the code base to see how to use it.